### PR TITLE
Release 0.25.2 + new update version script 🪄 

### PR DESCRIPTION
**Pull request by otaviopace**
This Pull Request does a few things:

- Updates the `NEWS.md`, cherry-picked from: https://github.com/graphprotocol/graph-node/pull/3317;
- Adds a new `scripts` folder with bash commands to change the `Cargo.toml`/`Cargo.lock` versions automatically;
- Updates the `Cargo.toml`/`Cargo.lock` files (done using the new script ✨ );
- Adds a check to the CI so that no new `crate` can be added to the repo without having the correct version (aka the same as the rest).

### CODE REVIEW

**Review by mangas — APPROVED** [review]

**Review by mangas — COMMENTED** [review]
LGTM but CI is not happy

**Comment by neysofu** [comment]
@otaviopace https://github.com/graphprotocol/graph-node/pull/3347/commits/5f957f13fb25cf796fc8ceae0bb3fc3d6a53f830

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -229,3 +229,17 @@ jobs:
         with:
           command: check
           args: --release
+
+  version-check:
```
Comment (neysofu) [inline .github/workflows/ci.yml:233]: :clap:

**Review by neysofu — APPROVED** [review]
LGTM, we just need to delete https://github.com/graphprotocol/graph-node/pull/3347/commits/5f957f13fb25cf796fc8ceae0bb3fc3d6a53f830!

**Comment by otaviopace** [comment]
@mangas @neysofu I've pushed a broken commit just to test the CI, I'll remove it, thanks for the reviews ❤️

**Review by tilacog — APPROVED** [review]

**Pull request merged**